resources/js/components/LiveEvents.vue

```diff
@@ -69,49 +75,67 @@ export default {
 	echo: {
 	    'ImageUploaded': (payload, vm) => {
 			vm.events.unshift({
-				type:      'image',
-				city:      payload.city,
-				state: 	   payload.state,
-				country:   payload.country,
-				imageName: payload.imageName
+				type: 'image',
+				city: payload.city,
+				state: payload.state,
+				country: payload.country,
+				imageName: payload.imageName,
+                teamName: payload.teamName,
+                countryCode: payload.countryCode
 			});
 		},
 		'NewCountryAdded': (payload, vm) => {
 			vm.events.unshift({
-				type:      'country',
-				country:   payload.country,
+				type: 'country',
+				country: payload.country,
 				countryId: payload.countryId
 			})
 		},
 		'NewStateAdded': (payload, vm) => {
 			vm.events.unshift({
-				type:    'state',
-				state:   payload.state,
+				type: 'state',
+				state: payload.state,
 				stateId: payload.stateId
 			})
 		},
 		'NewCityAdded': (payload, vm) => {
 			vm.events.unshift({
-				type:   'city',
-				city:   payload.city,
+				type: 'city',
+				city: payload.city,
 				cityId: payload.cityId
 			})
 		},
 		'UserSignedUp': (payload, vm) => {
 			vm.events.unshift({
-				type:   'new-user',
+				type: 'new-user',
 				now: payload.now
 			})
 		}
 	},
 	data ()
     {
 		return {
+            dir: '/assets/icons/flags/',
 			events: []
 		};
 	},
 	methods: {
 
+	    /**
+         * Return location of country_flag.png
+         */
+        countryFlag (iso)
+        {
+            if (iso)
+            {
+                iso = iso.toLowerCase();
```

Comment:
Why not put this into return instead of assign to `iso`?

Reply:
It was having trouble with returning straight away last time I checked. It was copy/paste from another file